vars/uploadIndexFile.groovy

```diff
@@ -2,11 +2,13 @@ void call(Map args = [:]) {
     def latestBuildData = ['latest': "${BUILD_NUMBER}"]
     writeJSON file: 'index.json', json: latestBuildData
 
-    echo "Uploading index.json to s3://${ARTIFACT_PRODUCTION_BUCKET_NAME}/${args.indexFilePath}"
+    withCredentials([string(credentialsId: 'jenkins-artifact-bucket-name', variable: 'ARTIFACT_BUCKET_NAME')]) {
+        echo "Uploading index.json to s3://${ARTIFACT_BUCKET_NAME}/${args.indexFilePath}"
```

Comment:
Hi @tianleh can you review this. It looked like a bug in echo statement before so I changed it.
Thanks!

Reply:
Thanks for the update.